hack/components/bump-multus.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+source hack/components/yaml-utils.sh
+source hack/components/git-utils.sh
+
+#here we do all the object specific parametizing
+function __parametize_by_object() {
+	for f in ./*; do
+		case "${f}" in
+			./ClusterRoleBinding_multus.yaml)
+				yaml-utils::update_param ${f} subjects[0].namespace  "{{ .Namespace }}"
+				echo '---' >> ${f}
+				;;
+		./ClusterRole_multus.yaml)
+				echo '---' >> ${f}
+				;;
+			./ServiceAccount_multus.yaml)
+				yaml-utils::update_param ${f} metadata.namespace  "{{ .Namespace }}"
+				echo '{{ if .EnableSCC }}' >> ServiceAccount_multus.yaml
+				echo '---' >> ${f}
+				;;
+			./DaemonSet_kube-multus-ds-amd64.yaml)
+				yaml-utils::update_param ${f} metadata.name  "multus"
+				yaml-utils::update_param ${f} metadata.namespace  "{{ .Namespace }}"
+				yaml-utils::update_param ${f} spec.template.spec.containers[0].image  "{{ .MultusImage }}"
+				yaml-utils::set_param ${f} spec.template.spec.containers[0].imagePullPolicy  "{{ .ImagePullPolicy }}"
+				yaml-utils::delete_param ${f} spec.template.spec.containers[0].volumeMounts[2]
+				yaml-utils::update_param ${f} spec.template.spec.volumes[0].hostPath.path  "{{ .CNIConfigDir }}"
+				yaml-utils::update_param ${f} spec.template.spec.volumes[1].hostPath.path  "{{ .CNIBinDir }}"
+				yaml-utils::delete_param ${f} spec.template.spec.volumes[2]
+				echo '---' >> ${f}
+				;;
+			./CustomResourceDefinition_network-attachment-definitions.k8s.cni.cncf.io.yaml)
+				echo '---' >> ${f}
+		esac
+	done
+}
+
+echo 'Bumping multus'
+MULTUS_URL=$(yaml-utils::get_component_url multus)
+MULTUS_COMMIT=$(yaml-utils::get_component_commit multus)
+MULTUS_REPO=$(yaml-utils::get_component_repo ${MULTUS_URL})
+
+TEMP_DIR=$(git-utils::create_temp_path multus)
+trap "rm -rf ${TEMP_DIR}" EXIT
+MULTUS_PATH=${TEMP_DIR}/${MULTUS_REPO}
+
+echo 'Fetch multus sources'
+git-utils::fetch_component ${MULTUS_PATH} ${MULTUS_URL} ${MULTUS_COMMIT}
+
+(
+	cd ${MULTUS_PATH}
+	mkdir -p config/cnao
+	cp images/multus-daemonset.yml config/cnao
+
+	echo 'Split manifest per object'
+	cd config/cnao
+	$(yaml-utils::split_yaml_by_seperator . multus-daemonset.yml)
+	rm multus-daemonset.yml
+	$(yaml-utils::rename_files_by_object .)
+
+	echo 'parametize manifests by object'
+	__parametize_by_object
+
+	cat <<EOF > 000-ns.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Namespace }}
+EOF
+
+	cat <<EOF > SecurityContextConstraints_multus.yaml
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: multus
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:{{ .Namespace }}:multus
+{{ end }}
+---
+EOF
+
+	echo 'rejoin sub-manifests to final manifests'
+	RBAC_YAML=001-rbac.yaml
+	touch ${RBAC_YAML}
```

Comment:
We don't need to unitfy them, CNAO will works with individual yamls files under data.

Reply:
yeah but don't we want to gather them to `rbac` and `multus` like it was before?